static/js/popovers.js

```diff
@@ -81,8 +81,10 @@ function init_email_clipboard() {
 }
 
 function load_medium_avatar(user, elt) {
-    const user_avatar_url = "avatar/" + user.user_id + "/medium";
+    const avatar_path = "avatar/" + user.user_id + "/medium?v=" + user.avatar_version;
+    const user_avatar_url = new URL(avatar_path, window.location.href);
     const sender_avatar_medium = new Image();
+
```

Comment:
stray new line?

Reply:
Not really, it just bugs me to read too many lines in a row without a break, so I tried to make a "block" with all the definitions separated from the rest. But I can take it out.